num2.js

```diff
@@ -0,0 +1,44 @@
++var start = document.getElementById("start");
++var pause = document.getElementById("pause");
++var reset = document.getElementById("reset");
+
++//3
++var seconds= 0;
++var timerId;
+
+
++  //2
++// start.addEventListener("click", function() {
++//   });
++//       pause.addEventListener("click", function() {
++//         });
++//             reset.addEventListener("click", function() {
++//
++//             });
+
```

Comment:
Try to only work in one file and remove code comments. Use git to keep track of what works and what doesn't.